deploy-manage/deploy/cloud-enterprise/ece-install-offline-images.md

```diff
@@ -10,22 +10,18 @@ mapped_pages:
 
 Versions of the Elastic Stack, containing Elasticsearch, Kibana, and other products, are available as downloadable Docker images.
 
-The first table contains the stack versions that shipped with the 3.8 version of Elastic Cloud Enterprise. You can also check the [most recent stack packs and Docker images](#ece-recent-download-list), which might have released after the 3.8 version of ECE, as well as the [full list of available stack packs and Docker images](#ece-full-download-list).
+The first table contains the stack versions that shipped with the 4.0 version of Elastic Cloud Enterprise. You can also check the [most recent stack packs and Docker images](#ece-recent-download-list), which might have released after the 4.0 version of ECE, as well as the [full list of available stack packs and Docker images](#ece-full-download-list).
 
-| Docker images included with Elastic Cloud Enterprise 3.8.1 |
+| Docker images included with Elastic Cloud Enterprise 4.0.0 |
 | --- |
-| docker.elastic.co/cloud-enterprise/elastic-cloud-enterprise:3.8.1 |
-| docker.elastic.co/cloud-assets/elasticsearch:6.8.23-0 |
-| docker.elastic.co/cloud-assets/kibana:6.8.23-0 |
-| docker.elastic.co/cloud-assets/apm:6.8.23-0 |
-| docker.elastic.co/cloud-assets/elasticsearch:7.17.27-0 |
-| docker.elastic.co/cloud-assets/kibana:7.17.27-0 |
-| docker.elastic.co/cloud-assets/apm:7.17.27-0 |
-| docker.elastic.co/cloud-assets/enterprise-search:7.17.27-0 |
-| docker.elastic.co/cloud-release/elasticsearch-cloud-ess:8.17.1 |
-| docker.elastic.co/cloud-release/kibana-cloud:8.17.1 |
-| docker.elastic.co/cloud-release/elastic-agent-cloud:8.17.1 |
-| docker.elastic.co/cloud-release/enterprise-search-cloud:8.17.1 |
+| docker.elastic.co/cloud-enterprise/elastic-cloud-enterprise:4.0.0 |
+| docker.elastic.co/cloud-assets/elasticsearch:8.18.0-0 |
+| docker.elastic.co/cloud-assets/kibana:8.18.0-0 |
+| docker.elastic.co/cloud-assets/apm:8.18.0-0 |
+| docker.elastic.co/cloud-assets/enterprise-search:8.18.0-0 |
```

Comment:
These should also use `cloud-release` (`cloud-assets` and `-0` is only used in 7.x`)

```suggestion
| docker.elastic.co/cloud-release/elasticsearch-cloud-ess:8.18.0 |
| docker.elastic.co/cloud-release/kibana-cloud:8.18.0 |
| docker.elastic.co/cloud-release/elastic-agent-cloud:8.18.0 |
| docker.elastic.co/cloud-release/enterprise-search-cloud:8.18.0 |
```

Reply:
Probably the same is true for all the other files below that still use `cloud-assets` with 8.18.0.

(The image names are also different between 7.x and 8.x)